<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>upch_logo.png</normaloff>upch_logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(76, 76, 76);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="graph_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-color: rgb(27, 38, 30);
background-color: rgb(245, 245, 245);
border-style:outset;
border-radius: 2px;
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>79</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(6, 5, 50);
font: 8pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(167, 190, 203);
border-style:outset;
border-radius: 4px;
border-width: 1px;
padding: 2px;
border-color: rgb(34, 48, 38);
}</string>
    </property>
    <property name="text">
     <string> Slice:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>82</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="text">
     <string>Escoger slice</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="t_vol_h">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(167, 190, 203);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1px;
padding: 2px;
border-color: rgb(34, 48, 38);
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe MDL2 Assets'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="graph_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-color: rgb(27, 38, 30);
background-color: rgb(245, 245, 245);
border-style:outset;
border-radius: 2px;
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_vol_h">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(6, 5, 50);
font: 8pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(167, 190, 203);
border-style:outset;
border-radius: 4px;
border-width: 1px;
padding: 2px;
border-color: rgb(34, 48, 38);
}</string>
    </property>
    <property name="text">
     <string>Volumen Hipocampo:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_file">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="text">
     <string>Guardar archivos</string>
    </property>
   </widget>
   <widget class="QComboBox" name="load_file">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="minimumContentsLength">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Cargar NIfTI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cargar DICOM</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="play_both">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="text">
     <string>Reproducir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_both">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_both">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_op">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="text">
     <string>Iniciar Cálculo Volumétrico</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_both">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="t_frame">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(167, 190, 203);
border-style:outset;
border-radius: 4px;
border-width: 1px;
padding: 2px;
border-color: rgb(34, 48, 38);
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>23</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font: 22.5pt &quot;MS Reference Sans Serif&quot;;
color: rgb(221, 230, 237);
background-color:rgb(82, 109, 130);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 15px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}
</string>
    </property>
    <property name="text">
     <string> HippoSeg</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_info">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(6, 5, 50);
font: 9pt &quot;Segoe MDL2 Assets&quot;;
background-color:rgb(221, 230, 237);
border-style:outset;
border-top-left-radius: 4px;
border-radius: 4px;
border-width: 1.5px;
padding: 2px;
border-color: rgb(27, 38, 30);
}</string>
    </property>
    <property name="text">
     <string>Información (?)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_carrera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingeniería Biomédica - 2023</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo_upch">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>upch_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo_pucp">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>530</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pucp_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
